c2_components/src/mfx_c2_decoder_component.cpp

```diff
@@ -1404,7 +1404,7 @@ mfxStatus MfxC2DecoderComponent::DecodeFrame(mfxBitstream *bs, MfxC2FrameOut&& f
                         frame_out = *found;
                     } else {
                         MFX_DEBUG_TRACE_STREAM("Not found LOCKED!!!");
```

Comment:
Actually, this can never happen here. Anyway, let's return this fatal error here in case.  
mfx_sts = MFX_ERR_UNKNOWN;

Reply:
ok